package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "resourcemod-server",
+  "version": "1.0.0",
+  "author": "",
+  "license": "GPL-3.0",
+  "scripts": {
+    "build": "tsc"
```

Comment:
I'd call it compile or even better - make it just like in docs to avoid questions "Why npm run ts:compile is not working but the docs says it have to work"

Reply:
Up to you on this one `build` is what people usually call it.
Also updated the PR on the docs to `build` that's why here is also `build`

Reply:
https://github.com/resourcemod/documentation/pull/3/files